google/cloud/security/scanner/audit/bigquery_rules_engine.py

```diff
@@ -0,0 +1,284 @@
+# Copyright 2017 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Rules engine for Big Query data sets"""
+from collections import namedtuple
+import itertools
+import re
+
+# pylint: disable=line-too-long
+from google.cloud.security.common.gcp_type import bigquery_access_controls as bq_acls
+# pylint: enable=line-too-long
+from google.cloud.security.common.util import log_util
+from google.cloud.security.scanner.audit import base_rules_engine as bre
+from google.cloud.security.scanner.audit import errors as audit_errors
+
+LOGGER = log_util.get_logger(__name__)
+
+
+# TODO: move this to utils since it's used in more that one engine
+def escape_and_globify(pattern_string):
```

Comment:
This method seems to be used in multiple places now.  
https://github.com/GoogleCloudPlatform/forseti-security/search?utf8=%E2%9C%93&q=escape_and_globify&type=

So, can we refactor this to a common util method?  How about a regex_util.py and put it here?
https://github.com/GoogleCloudPlatform/forseti-security/tree/master/google/cloud/security/common/util